src/itest/resources/features/corporate_disqualification.feature

```diff
@@ -0,0 +1,24 @@
+Feature: Process corporate disqualified officer information
+
+  Scenario Outline: Processing corporate disqualified officer information successfully
+
+    Given disqualified officers data api service is running
+    When I send corporate PUT request with payload "<data>" file
+    Then I should receive 200 status code
+    And the CHS Kafka API is invoked successfully with "<officerId>"
+
+    Examples:
+      | data                                  | officerId     |
+      | corporate_disqualified_officer        | 1234567891    |
```

Comment:
Is using a scenario outline with example necessary if you only have one example case. You could do this instead?
```suggestion
  Scenario: Processing corporate disqualified officer information successfully

    Given disqualified officers data api service is running
    When I send corporate PUT request with payload corporate_disqualified_officer file
    Then I should receive 200 status code
    And the CHS Kafka API is invoked successfully with 1234567891
```

Reply:
I think it's best to keep these files generic to make them easier to add to in future even if it means having a single example per scenario for the time being.

Reply:
Makes sense